ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -20,7 +20,119 @@
       </ul>
     </nav>
   </div>
-  {{! overview cards here }}
+  <Toolbar>
+    <ToolbarActions>
+      <ToolbarLink @route="secrets.destinations.create" @type="add" data-test-create-destination>
+        Create new destination
+      </ToolbarLink>
+    </ToolbarActions>
+  </Toolbar>
+
+  <OverviewCard @cardTitle="Secrets by destination" class="has-top-margin-l">
+    {{#if this.fetchAssociationsForDestinations.isRunning}}
+      <div data-test-sync-overview-loading>
```

Comment:
I thought we talked about using ` <LayoutLoading />` here but perhaps I'm misremembering our design meeting